{
	"id": "CVE-2018-14449",
	"description": "An issue was discovered in libgig 4.1.0. There is an out of bounds read in gig::File::UpdateChunks in gig.cpp.",
	"debian_bug": 931309,
	"scope": "local",
	"packages": [
		{
			"name": "libgig",
			"status": "open",
			"urgency": "unimportant",
			"repository": [
				{
					"name": "trixie",
					"version": "4.4.1-1"
				}
			]
		}
	]
}
